graph_dfs_debug/Answers.md

```diff
@@ -1 +1,109 @@
 Describe the fixes/improvements you made to the Graph implementation here.
+
+
+Hi Friend!
+I was able to review your graph_dfs files and I think I can help. Let's go over the Issues list that you provided and find out what's going on.
+
+ISSUES LOG:
+1. Nothing seems to connect, my edges aren't showing up.
+	If you look at your graph.py / def_add_edges function, it seems that vertices begins at "start", but connects to "start". Also, your bidirectional if statement says "end" to "end".  I changed it to say the following:
+
+    def add_edge(self, start, end, bidirectional=True):
+        self.vertices[start].add(start) //change to self.vertices[start].add(end)
+        if bidirectional:
+            self.vertices[end].add(start) // self.vertices[end].add(start)
+********************************************************
+
+2. All the vertexes are the same color.  They're supposed to be different colors
```

Comment:
Does this solution keep connected components the same color?

Reply:
I don't think so.  But, I was able to make the corrections.